.github/workflows/ci.yml

```diff
@@ -261,9 +273,16 @@ jobs:
         if: github.ref == 'refs/heads/main' && env.GHCR_USERNAME != ''
         uses: sigstore/cosign-installer@v2.5.1
 
-      - name: Sign Artifacts to main release channel
+      - name: Sign Artifacts to main release channel for Distroless
+        if: github.ref == 'refs/heads/main' && env.GHCR_USERNAME != ''
+        run: make build/distroless/docker.sign
```

Comment:
We need to change the Makefile as well. I'm not sure what the best approach is, i believe we should have one single make target `make docker.build/promote/sign/..` that knows which images to build. This way we probably don't even need to change the CI workflows :thinking:

Reply:
If the goal is to keep single Makefile targets for all supported images, this Makefile portion (plus others...) needs to be changed:

https://github.com/external-secrets/external-secrets/blob/ed0ceb8d8437c3302fd2e0e6a622ab16bcbe9973/Makefile#L207-L210

A `docker build` command is needed for each image / Dockerfile.

If the goal is to keep all Dockerfiles in the project's root directory (like today - there are two Dockerfiles: `Dockerfile` and `Dockerfile.standalone`), you can create `Dockerfile.ubi` and place it into the root directory as well.

With that approach, a additional `docker build` is needed (plus other commands for the UBI-based image):

```sh
```